src/transaction/transaction_group_complete_manager.cpp

```diff
@@ -101,7 +101,29 @@ namespace cubtx
   }
 
   //
-  // close_current_group close the current group. Next coming transactions will be added into the next group.
+  // set_current_group_minimum_transactions set minimum number of transactions for current group.
+  //
+  complete_manager::id_type group_complete_manager::set_current_group_minimum_transactions (
+	  int count_minimum_transactions,
+	  bool &has_group_enough_transactions)
+  {
+    std::unique_lock<std::mutex> ulock (m_group_mutex);
+    m_current_group_min_transactions = count_minimum_transactions;
+
+    if (m_current_group_min_transactions <= m_current_group.get_container ().size ())
```

Comment:
How the number of transactions in group container can be greater than m_current_group_min_transactions  ?

Reply:
For this case it should be equal. More generally, we can set (with parameter) the minimum number of transactions in group (for performance). This is the purpose of "<=".

Reply:
What happens in a slave node if group exceeds m_current_group_min_transactions? Is this case accepted?